.github/pull_request_template.md

```diff
@@ -1,42 +1,17 @@
 ## Requirements
 
 - [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
-- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
-- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).
-
+- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
```

Comment:
@brandones  Should we also include a requirement of having test for each PR?

Reply:
Great idea @donaldkibet ! I'll add that